games/p.yaml

```diff
@@ -438,6 +438,24 @@
   updated: 2015-04-03
   url: http://quakeone.com/proquake/
 
+- name: proxx
+  development: complete
+  images:
+  - https://user-images.githubusercontent.com/234957/57080494-8caff680-6cea-11e9-8477-54ea0e1670e2.png
+  repo: https://github.com/GoogleChromeLabs/proxx
+  url: https://proxx.app/
+  license:
+  - Apache
+  status: playable
+  framework: WebGL
+  lang:
+  - TypeScript
+  - JavaScript
```

Comment:
```suggestion
```
looks like the only js files in the project are config/packaging, so it's not as important to mention them

Reply:
Ah fair point thanks will fix that :)